js/src/parse.js

```diff
@@ -114,13 +126,8 @@ function getLiteralValue(value, token) {
 /**
  * 
  * @param {string} text The text to parse.
- * @param {Object} [options] The options object.
- * @param {string} [options.mode="json"] The parsing mode.
- * @param {boolean} [options.ranges=false] Determines if ranges will be returned
- *      in addition to `loc` properties.
- * @param {boolean} [options.tokens=false] Determines if tokens are returned in
```

Comment:
maybe the documentation here is obvious enough that it should be dropped, but if not, it should move to the members of MomoaParseOptions